connection-service/new-app-gateway/azuredeploy.json

```diff
@@ -65,6 +65,19 @@
         "description": "Application Gateway Name including the unique CS suffix"
       }
     },
+    "enableExternalAccess": {
+      "type": "bool",
+      "metadata": {
+          "description": "'true' if a Public IP address should be created, otherwise 'false'."
+      }
+    },
+    "enableRadiusMfa": {
```

Comment:
In this file I don't think it has anything to do with MFA, it's more about enabling external access to the management interface? I'd suggest renaming.

Reply:
good point